runtime/indent/stylus.vim

```diff
@@ -0,0 +1,129 @@
+" Vim indent file
+" Language: Stylus
+" Maintainer: Marc Harter
+" Last Change: 2010 May 21
+" Based On: sass.vim from Tim Pope
+"
+if exists("b:did_indent")
+  finish
+endif
+unlet! b:did_indent
+let b:did_indent = 1
+
+setlocal indentexpr=GetStylusIndent()
+setlocal indentkeys=o,O,*<Return>,},],0),!^F
+setlocal formatoptions+=r
```

Comment:
missing undo_indent var

Reply:
Not sure, if I implemented it the right way.

Reply:
no, similar to `b:undo_ftplugin` you should undo the `indent` settings:  `:let b:undo_indent = 'setl indentexpr< indentkeys< '